printer_mods/README.md

```diff
@@ -416,4 +416,5 @@ like so:
 | | [V0 1515 T-nut](./Zen3D/V0_1515_T-nut)| M3 square and hex nuts for Voron0 1515 frame | :heavy_check_mark: |:x: |:x: |:x:|
 | zruncho | [V0 No-Drop Nuts](./zruncho/V0_No_Drop_Nuts) | Voron0 nut inserts which auto-align (LDO frames only) | :heavy_check_mark: | :x: | :x: | :x: |
 | ZZP8202 | [Pull-out electrical store](./ZZP8202/Pull-out_electrical_store) | Voron 2.4 Pull-out electrical store（350mm only） | :x: | :x: | :heavy_check_mark: | :x: | :x: |
+| ZZP8202 | [S4 Filament motion sensor](./ZZP8202/S4_Filament_motion_sensor) | A Filament motion sensor originated from VORON M4 Extruder| :grey_question: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
```

Comment:
Your username should be left blank here, since it's already on the line above.

Reply:
I have update the Readme and deleted the excel file.  Should I sync fork before you approve the pull request?

Reply:
Unfortunately I can only do reviews, I cannot merge the PR.  It may still take some time before it can be merged, but it's always a good idea to keep your fork up to date.

Reply:
Get it, thank you.